tests/spread/extensions/ros-meta-hello/task.yaml

```diff
@@ -0,0 +1,68 @@
+summary: Build and run a basic ROS snap using meta-ros extension
+
+kill-timeout: 180m
+
+environment:
+
+  SNAPCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: "1"
+
+  SNAP: catkin-noetic-hello
+  SNAP_DIR: "../../plugins/v2/snaps/$SNAP"
+
+  INTERFACE: ros-noetic
+
+  META_SNAP/catkin_noetic_ros_core: ros-noetic-ros-core
+  EXTENSION/catkin_noetic_ros_core: ros1-noetic-ros-core
+
+  META_SNAP/catkin_noetic_ros_base: ros-noetic-ros-base
+  EXTENSION/catkin_noetic_ros_base: ros1-noetic-ros-base
+
+  META_SNAP/catkin_noetic_robot: ros-noetic-robot
+  EXTENSION/catkin_noetic_robot: ros1-noetic-robot
+
+  META_SNAP/catkin_noetic_desktop: ros-noetic-desktop
+  EXTENSION/catkin_noetic_desktop: ros1-noetic-desktop
+
+# The content snap required for the test to succeed is only
+# available on a subset of all the architectures this testbed
+# can run on.
+systems:
+  - ubuntu-20.04
+  - ubuntu-20.04-64
+  - ubuntu-20.04-amd64
+  - ubuntu-20.04-arm64
```

Comment:
This means that we must push the content-sharing snap for arm. Thus, we will need the remote build to be working for that. How are we going to automate this since remote-build require a launchpad authentication file to be available.
If we want to use our own ARM server, then we will need to have this CI available publicly I guess.
(This is slightly out of the scope of the MR)